shared/login/forms/container.desktop.js.flow

```diff
@@ -0,0 +1,9 @@
+/* @flow */
+import React from 'react'
+
+export type Props = {
+    onBack: () => void
+};
+
```

Comment:
True, but as far as the thing that imports it, it doesn't know the difference between a stateless functional component and component that is made from a class.